core/scripts/update-flow-fixtures.js

```diff
@@ -146,6 +147,9 @@ async function generateFlowResult() {
 
   // Normalize some data so it doesn't change on every update.
   for (const {lhr} of flowResult.steps) {
+    cleanAndFormatLHR(lhr, {skipDescription: true});
+    // TODO: should remove this next line if cleanAndFormatLHR is refactored
+    lhr.configSettings.auditMode = false;
```

Comment:
`cleanup-LHR-for-diff` is already kind of shaky but it's at least focused; it doesn't feel great to turn it into a utility (as evidenced by this TODO :). 

`assetSaver.normalizeTimingEntries` is already a thing from https://github.com/GoogleChrome/lighthouse/pull/13715#discussion_r818206038, seems easy to do the same this for this?